src/lib/utils/__tests__/fetcher.test.ts

```
@@ -0,0 +1,458 @@
+        body: JSON.stringify({ data: 'test' }),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+      })
+    })
+
+    it('カスタムヘッダーを正しくマージする', async () => {
+      process.env.NEXT_PUBLIC_DEV_MODE = 'true'
+
+      mockFetch.mockResolvedValue({
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)